framework/components/ecl-breadcrumbs/_breadcrumbs.scss

```diff
@@ -1,268 +1,55 @@
-/* stylelint-disable */
+/*
+ * Breadcrumbs
+ * @define breadcrumbs
+ */
+
 $breadcrumb-heigth: 2.778em;
 
-.breadcrumb {
-  position: relative;
-  margin: 0;
-  padding: 0;
-  background: none;
-  font-size: 0.833em;
-  font-weight: 600;
+.ecl-breadcrumbs {
+  @include ecl-hidden-print();
+
   border-bottom: 1px solid #fff;
-  border-radius: 0;
+  font-size: map-get($ecl-font-size, 's');
   line-height: normal;
+  margin: 0;
+  padding: 0;
+}
 
-  @include ecl-hidden-print();
-
-  ol,
-  ul {
-    list-style: none;
-    max-width: none;
-    margin: 0;
-    padding: 0;
-  }
+.ecl-breadcrumbs__segments-wrapper {
+  list-style: none;
+  margin: 0;
+  max-width: none;
+  padding: 0;
 }
 
-.breadcrumb__segment {
-  display: none;
-  position: relative;
-  width: 100%;
-  margin: 1em 0 0.8em;
+.ecl-breadcrumbs__segment {
+  display: inline-block;
+  margin: map-get($ecl-spacing, 's') 0 map-get($ecl-spacing, 'xxs');
 }
 
-a.breadcrumb__link {
+.ecl-breadcrumbs__link {
+  color: #fff;
   display: inline-block;
-  max-width: 100%;
-  margin-right: 8px;
   font-weight: bold;
-  color: #fff;
+  margin-right: map-get($ecl-spacing, 'xxxs');
   text-decoration: none;
 
+  // Add the ">" icon
   &::after {
-    @extend .breadcrumb__link-arrow;
+    @extend %ecl-icon--after;
```

Comment:
I'm wondering if we could get rid of those `@extend %ecl-icon--xxx` and include them in the `ecl-icon` mixin... What do you think?